cityofaustin/joplin
# 4169 import department pages

# Description
Super basic department page importing

### what this does
* Imports department pages
* Imports department directors
### what this doesn't
* Import/associate top pages/related pages
* import images
* associate department pages with department groups








# Types of changes



# How can this be tested?



 
check out the PR app (https://joplin-pr-4169-import-departme.herokuapp.com/admin/pages/search/) and see the imported department

# Checklist:
